tests/exemplars.ttl

```diff
@@ -55,3 +55,18 @@ kb:QualityValueAttributionSituation-d760ff9f-8b16-4628-9236-fbe77b372e27
 	gufo:concernsQualityType kb:QualityType-3a0ff32d-9743-46e4-bdcd-e86777f98551 ;
 	.
 
+kb:Type-15690ddc-002a-48e6-8e2f-e0fafaf5bb3a
```

Comment:
I'd like to see the purpose of these individuals; I can't see them being applied somewhere.